### DISCUSSION TITLE
Ability to hide menu bar icon (with idea for how)

### DISCUSSION THREAD

**Discussion by jonnyborbs** [will-stone/browserosaurus#653] [Ideas]
I know this came up once or twice before and it was suggested to use 3rd party apps to hide the icon. I know that this tool is largely interface-less and having a way to bring up the config dialog is necessary, but why can't I hide the icon, while the application remains in memory - then if I run the app again manually, the config window comes up? The configuration steps/flow is very, very secondary to the normal "all set, clicking on URLs is my life now" workflow, but the presence of the menu bar icon sacrifices valuable space and creates distraction for the sake of it.

Particularly annoying now that the new MacBooks have notches, and icons refuse to progress past the notch. I'd like to suggest that this new hardware design is enough to warrant revisiting the feature request.

Thanks so much!

**Comment by will-stone** [comment]
Sounds great, I'd welcome a PR with this functionality. I reckon you'd want to include a little onboarding message when it first opens to explain how to open the config again. I look forward to seeing your submission.

Until then "it was suggested to use 3rd party apps to hide the icon".

Thanks 😄
